### PR TITLE
[DOC] Clarify release notes for Parquet version

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Corrects the release notes: 
* Alex Strate commented in the Community Slack that the vParquet4 section of the 2.6 release notes was confusing: vParquet is only usable with Temp[o 2.6 instead of the other way around. 

This PR incorporates Joe Elliott's response into the release notes and the upgrade.md documentation. 

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/release-notes/v2-6.md [hunk 1]
```diff
@@ -168,7 +168,11 @@ vParquet4 format is now the default block format.
 It's production ready and we highly recommend switching to it for improved query performance. [PR [3810](https://github.com/grafana/tempo/pull/3810)]
 
 Upgrading to Tempo 2.6 modifies the Parquet block format.
-Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use Tempo 2.6 with vParquet3.
+You don't need to do anything with Parquet to go from 2.5 to 2.6.
+All of your old blocks remain vParquet3, which 2.6 reads.
+Tempo 2.6 creates vParquet4 blocks by default, which enables the new TraceQL features.
+
+Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use vParquet4 with Tempo 2.6 and newer.
```
Comment (ie-pham) [inline docs/sources/tempo/release-notes/v2-6.md:175]: vParquet4 was already available as an "experimental" block format in 2.5 and became the default in 2.6 so users can already use vParquet4 in Tempo 2.5 - but they would need to upgrade to 2.6 to get all the new traceql features

File: docs/sources/tempo/release-notes/v2-6.md [hunk 2]
```diff
@@ -168,7 +168,11 @@ vParquet4 format is now the default block format.
 It's production ready and we highly recommend switching to it for improved query performance. [PR [3810](https://github.com/grafana/tempo/pull/3810)]
 
 Upgrading to Tempo 2.6 modifies the Parquet block format.
-Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use Tempo 2.6 with vParquet3.
+You don't need to do anything with Parquet to go from 2.5 to 2.6.
+All of your old blocks remain vParquet3, which 2.6 reads.
```
Comment (ie-pham) [inline docs/sources/tempo/release-notes/v2-6.md:172]: The default version of vParquet in Tempo 2.5 was vParquet3 but it is possible some users chose to use vParquet2

File: docs/sources/tempo/setup/upgrade.md [hunk 3]
```diff
@@ -84,7 +84,11 @@ vParquet4 format is now the default block format.
 It's production ready and we highly recommend switching to it for improved query performance. [PR [3810](https://github.com/grafana/tempo/pull/3810)]
 
 Upgrading to Tempo 2.6 modifies the Parquet block format.
-Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use Tempo 2.6 with vParquet3.
+You don't need to do anything with Parquet to go from 2.5 to 2.6.
```
Comment (ie-pham) [inline docs/sources/tempo/setup/upgrade.md:87]: same comments as the release notes

File: docs/sources/tempo/release-notes/v2-6.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/tempo/release-notes/v2-6.md:172]:
```suggestion
If you used vParquet2 or vParquet3, all of your old blocks remain and are readable by version 2.6.
```

File: docs/sources/tempo/release-notes/v2-6.md (hunk 1 above)
Comment (knylander-grafana) [reply docs/sources/tempo/release-notes/v2-6.md:175]: With the other update, we can remove part of this sentence. 
```suggestion
Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use vParquet4 with Tempo 2.5 and newer.
If you are using 2.5 with vParquet4, you'll need to upgrade to Tempo 2.6 to use the new TraceQL features. 
```

File: docs/sources/tempo/setup/upgrade.md [hunk 4]
```diff
@@ -84,7 +84,11 @@ vParquet4 format is now the default block format.
...
+All of your old blocks remain vParquet3, which 2.6 reads.
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/upgrade.md:88]:
```suggestion
If you used vParquet2 or vParquet3, all of your old blocks remain and can be read by Tempo 2.6.
```

File: docs/sources/tempo/setup/upgrade.md [hunk 5]
```diff
@@ -84,7 +84,11 @@ vParquet4 format is now the default block format.
...
+Tempo 2.6 creates vParquet4 blocks by default, which enables the new TraceQL features.
+
+Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use vParquet4 with Tempo 2.6 and newer.
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/upgrade.md:91]:
```suggestion
Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use vParquet4 with Tempo 2.5 and newer.
If you are using 2.5 with vParquet4, you'll need to upgrade to Tempo 2.6 to use the new TraceQL features.
```

File: docs/sources/tempo/setup/upgrade.md (hunk 3 above)
Comment (knylander-grafana) [reply docs/sources/tempo/setup/upgrade.md:87]: Thank you for reading this! I'm glad you read through this.

**Comment by knylander-grafana** [comment]
@ie-pham Updated the statements. What do you think?

**Review by ie-pham — APPROVED** [review]
Perfect!

**Comment by knylander-grafana** [comment]
Git rebase went wrong. Recreated this change here - https://github.com/grafana/tempo/pull/4135

**Pull request closed without merging**